### PR TITLE
OP-7469 : Mapping URLs for Export to csv 

### PR DESCRIPTION
**Pull request by sriharshakancharla**
https://devopsmx.atlassian.net/browse/OP-7469

We prefer small, well-tested pull requests.

Please refer to [Contributing to Spinnaker](https://spinnaker.io/community/contributing/).

When filling out a pull request, please consider the following:

* Follow the commit message conventions [found here](https://spinnaker.io/community/contributing/submitting/).
* Provide a descriptive summary for your changes.
* If it fixes a bug or resolves a feature request, be sure to link to that issue.
* Add inline code comments to changes that might not be obvious.
* Squash your commits as you keep adding changes.
* Add a comment to @spinnaker/reviewers for review if your issue has been outstanding for more than 3 days.

Note that we are unlikely to accept pull requests that add features without prior discussion. The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.

### CODE REVIEW

**Comment by opsmxuser** [comment]
This PR  triggered this build in this execution, please merge to master only if this succeeds:  https://spin.prod.opsmx.com/#/applications/gitprtrigger/executions/details/01FGR5DJQQY5PP4ND4C4ZR42C6

File: gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/OpsmxPlatformController.groovy [hunk 1]
```diff
@@ -119,6 +125,26 @@ class OpsmxPlatformController {
     return opsmxPlatformService.getPlatformResponse7(version, type, source, source1, source2, source3, source4)
   }
 
+  @ApiOperation(value = "Endpoint for Insights controller to download csv file")
+  @GetMapping(value = "v1/insights/download", produces = "text/csv")
```
Comment (Pranav-b-7) [inline gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/OpsmxPlatformController.groovy:129]: Make v1 in the endpoint as path variable . Something like this : {version}

File: gate-web/src/main/groovy/com/opsmx/spinnaker/gate/controllers/OpsmxAuditClientServiceController.groovy [hunk 2]
```diff
@@ -114,4 +120,28 @@ class OpsmxAuditClientServiceController {
 
     return opsmxAuditClientService.getAuditClientResponse7(version, type, source, source1, source2, source3, source4, source5)
   }
+
+  @ApiOperation(value = "Endpoint for Insights controller to download csv file")
+  @RequestMapping(value = "/v1/users/{username}/{source}/download", produces = "text/csv", method = RequestMethod.GET)
```
Comment (Pranav-b-7) [inline gate-web/src/main/groovy/com/opsmx/spinnaker/gate/controllers/OpsmxAuditClientServiceController.groovy:125]: Make v1 in the endpoint as path variable . Something like this : {version}

**Review by Pranav-b-7 — CHANGES_REQUESTED** [review]

File: gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/OpsmxPlatformController.groovy (hunk 1 above)
Comment (sriharshakancharla) [reply gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/OpsmxPlatformController.groovy]: modified code

File: gate-web/src/main/groovy/com/opsmx/spinnaker/gate/controllers/OpsmxAuditClientServiceController.groovy (hunk 2 above)
Comment (sriharshakancharla) [reply gate-web/src/main/groovy/com/opsmx/spinnaker/gate/controllers/OpsmxAuditClientServiceController.groovy]: modified code

**Comment by opsmxuser** [comment]
This PR  triggered this build in this execution, please merge to master only if this succeeds:  https://spin.prod.opsmx.com/#/applications/gitprtrigger/executions/details/01FGR7WQ4VSWWT88QMDQ549G3V

**Review by Pranav-b-7 — APPROVED** [review]

**Pull request merged**